pkg/osx/req_ext.txt

```diff
@@ -0,0 +1,4 @@
+cryptography==1.1.2
+pyOpenSSL==0.15.1
+pygit2==0.23.2
+
```

Comment:
These three require different parameters to install than the rest. They depend on openssl, libgit2, etc that require specific paths be passed as parameters.